randovania/data/json_data/prime3.txt

```diff
@@ -3881,13 +4745,13 @@ Asset id: 13589229454537797972
 
```

Comment:
Above, change Command Vault to Morph Ball Door and rename to "Morph Ball Door to Command Vault". Connect it to Flux Control as Trivial